configs/raspberrypi_64_config

```diff
@@ -0,0 +1,111 @@
+# Binaries generated with the following script:
+#    https://github.com/mendersoftware/mender-convert-integration-scripts/blob/master/build-uboot-rpi64.sh
+
+ASSETS_PATH="rpi_arm64"
+
+# in the Aarch64 case, it seems to always be "kernel8.img"
+RASPBERRYPI_KERNEL_IMAGE="kernel8.img"
+
+RASPBERRYPI_BINARIES="raspberrypi_arm64-2024.04.tar.gz"
+RASPBERRYPI_BINARIES_URL="${MENDER_STORAGE_URL}/mender-convert/uboot/raspberrypi/${RASPBERRYPI_BINARIES}"
+
+RASPBERRYPI_OS_REQUIRED_ID="debian"     # also applies to the "Raspberry Pi OS" images
+RASPBERRYPI_OS_MINIMAL_VERSION=12       # this is "bookworm" or later, as it changed the boot partition mount directory
+
+source configs/raspberrypi_common_config
+
+function platform_modify() {
+    if file -L work/rootfs/bin/sh | grep -qF "ELF 32-bit"; then
+        echo "This configuration requires a 64-bit image for Raspberry Pi.".
+        exit 1
+    fi
+
+    if ! check_image_os; then
+        echo "This configuration requires an image based on $RASPBERRYPI_OS_REQUIRED_ID, version $RASPBERRYPI_OS_MINIMAL_VERSION or later."
+        exit 1
+    fi
+    
+    mkdir -p work/rpi/binaries
+
+    if [ -f "assets/${RASPBERRYPI_BINARIES}" ]; then
+        run_and_log_cmd "cp assets/${RASPBERRYPI_BINARIES} work/rpi/binaries"
```

Comment:
Actually it will be removed, this is a tradeoff I decided to go for in order to provide a first usable branch to the community AND a preliminary PR for discussion. Once we have the binaries at a downloadable place, I think this should not be there - unless we decide to encourage people to inject their own binaries (which can be useful at times)